### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cawrites**
edit readme file. change for Git

Expanded steps on Readme file for creating a pull request with syntax for each command and recommendations for Git tools and editor Visual Studio Code.

Thanks for contributing this Pull Request. Make sure that you submit this Pull Request against the `master` branch of this repository, add a brief description, and tag the relevant issue(s) and PR(s) below.

- Relevant Issues : (compulsory)
- Relevant PRs : (optional)
- Type of change :
  - [ ] New feature
  - [ ] Bug fix for existing feature
  - [ ] Code quality improvement
  - [ ] Addition or Improvement of tests
  - [ x] Addition or Improvement of documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -195,12 +195,55 @@ Executed using lambdas created for deletion purposes, once the deletion is succe
...
 1. Fork Trapheus repo
-2. Make changes locally and add unit tests as needed
-3. Run the test suite in the repo to ensure existing flows are not breaking
-```
-cd Trapheus
-python -m pytest tests/ -v #to execute the complete test suite
-python -m pytest tests/unit/test_get_dbstatus_function.py -v #to execute any individual test
-```
-4. Raise a pull request from the fork
+1. Create a working branch.
```
Comment (namitad) [inline README.md:207]: Nit: please correct the numbering on the steps

**Review by namitad — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (cawrites) [reply README.md:207]: @namitad has this been resolved? The step that was added is out of order.

Comment (namitad) [reply README.md:207]: @cawrites all the steps added have the numbering 1. please rectify that in your PR

**Comment by cawrites** [comment]
I am not seeing this in preview:
https://github.com/intuit/Trapheus/blob/a28884129306cb914f252f92898cf085794b6be1/README.md

File: README.md (hunk 1 above)
Comment (cawrites) [reply README.md:207]: This is what's showing in the code already:
1. Fork Trapheus repo
1. Create a working branch.
    ```bash
    git branch trapheus-change1
    ```
1. Confirm working branch for changes.
   ```bash
    git checkout trapheus-change1
   ```

Comment (cawrites) [reply README.md:207]: The PR currently shows:
1. Fork Trapheus repo
1. Create a working branch.
    ```bash
    git branch trapheus-change1
    ```
1. Confirm working branch for changes.
   ```bash
    git checkout trapheus-change1
   ```
You can combine boths command by typing `git checkout -b trapheus-change1`.
1. Make changes locally using an editor and add unit tests as needed.
1. Run the test suite in the repo to ensure existing flows are not breaking.
    ```bash
       cd Trapheus
       python -m pytest tests/ -v #to execute the complete test suite
       python -m pytest tests/unit/test_get_dbstatus_function.py -v #to execute any individual test
    ```
1. Stage edited files.
   ```bash
      git add contentfile.md 
    ```
    Or use `git add . ` for multiple files.

1. Commit changes from staging.
    ```bash
       git commit -m "trapheus-change1"
    ```
1. Push new changes to github
    ```bash
       git push --set-upstream origin trapheus-change1
    ```
1. Verify status of branch
    ```bash
    git status
    ```
    Review `Output` to confirm commit status.


1. Git push
    ```bash
        git push --set-upstream origin trapheus-change1
    ```

1. The `Output` will provide a link to create your Pull Request.

Comment (cawrites) [reply README.md:207]: image
I am not finding the reported issue.

Comment (cawrites) [reply README.md:207]: @namitad @stationeros please see Code Review page.

**Review by namitad — APPROVED** [review]

**Pull request merged**